browser/src/screenshot.ts

```diff
@@ -1,23 +1,52 @@
 import { type Page } from 'playwright'
 import * as gptscript from '@gptscript-ai/gptscript'
-import { getWorkspaceId } from './session.ts'
+import { getWorkspaceId, getGPTScriptEnv } from './session.ts'
 import { type IncomingHttpHeaders } from 'node:http'
+import { createHash } from 'node:crypto';
 
 const client = new gptscript.GPTScript();
+const ottoServerUrl = process.env.OTTO_SERVER_URL
 
-export async function screenshot (page: Page, headers: IncomingHttpHeaders): Promise<string> {
-  const workspaceId = getWorkspaceId(headers['x-gptscript-env'])
-  let screenshotName = `screenshot-${Date.now()}_${page.url().replace(/[^a-zA-Z0-9]/g, '_')}.png`
```

Comment:
Note: I changed this because it was producing invalid `downloadURLs` for certain pages and we're exposing the page URL as part of the `ScreenshotInfo` the tool now responds with.